src/RMBTControlServerCommunication.js

```diff
@@ -0,0 +1,100 @@
+let RMBTControlServerCommunication = (function () {
+    function RMBTControlServerCommunication() {
```

Comment:
All methods in here need the rmbtTestConfig. Could you make is a constructor parameter?